Gems/WhiteBox/Code/Source/EditorWhiteBoxComponent.cpp

```diff
@@ -239,10 +239,11 @@ namespace WhiteBox
     {
         if (m_renderMesh.has_value())
         {
-            (*m_renderMesh)->UpdateMaterial(m_material);
+            WhiteBoxMaterial material = m_material;
+            material.m_visible = true;
```

Comment:
I'm not sure about this - it could be related to the Editor visibility toggle in the Outliner I guess?

Reply:
There used to be a toggle button in the component card to quickly toggle the visibility of the WhiteBox mesh on/off. If we were to put this back in place, we would want to have the logic in `SetVisiblity` to do this with the `Atom` mesh and track the visibility state so it can be returned in `IsVisible`.

Reply:
Oh ok! I guess that can be discussed separately, and we can get ui-ux involved on whether a separate toggle is necessary or the visibility toggle in the outliner already fits the need.